classes/MusicAlbum/music_album.rb

```diff
@@ -0,0 +1,16 @@
+require_relative '../item'
+
+class MusicAlbum < Item
+  attr_accessor :on_spotify
+
+  def initialize(published_date = nil, on_spotify: false, archived: false)
+    super(published_date, archived: archived)
+    @on_spotify = on_spotify
```

Comment:
- [ ] Good job implementing `MusicAlbum` class, however, currently we are not saving data for the `music_album` and `genres`.  Kindly  make sure to save data after adding the `music album`  and `genres` as it is [required](https://github.com/microverseinc/curriculum-ruby/blob/main/group-capstone/catalog_of_my_things.md#:~:text=All%20data%20should%20be%20preserved%20by%20saving%20collections%20in%20.json%20files.) to preserve data 👍 

![Screenshot 2023-08-10 170806](https://github.com/yashodhicy/RubyCapstone/assets/57408419/d07f8373-e0b7-494a-8fba-1eeee9b12ee7)